src/network/core.rs

```diff
@@ -119,10 +119,15 @@ impl Core {
         ) {
             Ok(_) => (),
             Err(err) => {
+                // it seems something went wrong
+                // we must not leave dangling interfaces
+                // otherwise cleanup would become mess
+                // try removing leaking interfaces from host
+                let _ = core_utils::CoreUtils::remove_interface(&host_veth_name);
```

Comment:
Can you log the error if there is one, maybe as warning.

Reply:
sure